cmd/boulder-janitor/job.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	// errStat is a prometheus counter vector tracking the number of errors
+	// experienced by the janitor during operation sliced by a tabel label and a
```

Comment:
```suggestion
	// experienced by the janitor during operation sliced by a table label and a
```

One more tiny typo :-D

Reply:
omg I need to add `set :spell` to my `.vimrc` for Go files :sob:

Reply:
Fixed in 597c5ad